src/routes/authentication/Authentication.component.tsx

```diff
@@ -82,21 +114,57 @@ const Authentication = () => {
           setLoading(false);
           return;
         }
-
         navigate("/verification");
-        // axios.post formFields to DB here + remove log
-        console.log(data);
         setLoading(false);
         return;
       }
     );
   };
 
-  const handleSubmitVerification = (e: React.FormEvent<HTMLFormElement>) => {
+  const handleSubmitSignIn = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setLoading(true);
+    const authDetails = new AuthenticationDetails({
+      Username: formFields.email,
+      Password: formFields.password,
+    });
+
+    await user.authenticateUser(authDetails, {
+      onFailure(error) {
```

Comment:
This error handling isn't working for me, I think because you're missing a `setLoading(false)`